Packs/IdentityRecordedFuture/Classifiers/classifier-Recorded_Future_Identity_Incoming_Mapper.json

```diff
@@ -0,0 +1,327 @@
+{
+    "description": "",
+    "feed": false,
+    "id": "Recorded Future Identity - Incoming Mapper",
+    "mapping": {
+        "Recorded Future Identity Exposure": {
+            "dontMapEventToLabels": false,
+            "internalMapping": {
+                "Alert Category": {
```

Comment:
Didn't find this field om the layout - might be good idea to add it.

Reply:
Alert Category is the same for every identity alert and is included in title of alert, so we intentionally don't include it in the layout itself